Dockerfile

```diff
@@ -1 +1,14 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:20210701-2f4a4c7
```

Comment:
please update to the latest tag

Reply:
for the ark_base repo, there is no image with the latest tag

Reply:
there is now, just had to manually run the pipeline (as there were no changes).